features/trusted-types.yml

```diff
@@ -3,33 +3,6 @@ description: Trusted types allow you to lock down insecure parts of the DOM API
 spec: https://w3c.github.io/trusted-types/dist/spec/
 caniuse: trusted-types
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/2722
-# toJSON() is excluded because they are not needed to use trusted types in the
-# idiomatic way. toJSON() shipped in Chrome 90, and the rest in Chrome 83.
-compat_features:
-  - api.TrustedHTML
-  # - api.TrustedHTML.toJSON
-  - api.TrustedHTML.toString
-  - api.TrustedScript
-  # - api.TrustedScript.toJSON
-  - api.TrustedScript.toString
-  - api.TrustedScriptURL
-  # - api.TrustedScriptURL.toJSON
-  - api.TrustedScriptURL.toString
-  - api.TrustedTypePolicy
-  - api.TrustedTypePolicy.createHTML
-  - api.TrustedTypePolicy.createScript
-  - api.TrustedTypePolicy.createScriptURL
-  - api.TrustedTypePolicy.name
-  - api.TrustedTypePolicyFactory
-  - api.TrustedTypePolicyFactory.createPolicy
-  - api.TrustedTypePolicyFactory.defaultPolicy
-  - api.TrustedTypePolicyFactory.emptyHTML
-  - api.TrustedTypePolicyFactory.emptyScript
-  - api.TrustedTypePolicyFactory.getAttributeType
-  - api.TrustedTypePolicyFactory.getPropertyType
-  - api.TrustedTypePolicyFactory.isHTML
-  - api.TrustedTypePolicyFactory.isScript
-  - api.TrustedTypePolicyFactory.isScriptURL
-  - api.trustedTypes
-  - http.headers.Content-Security-Policy.require-trusted-types-for
-  - http.headers.Content-Security-Policy.trusted-types
+status:
+  compute_from:
+    - api.trustedTypes
```

Comment:
Hmm. On the one hand, I'd say that I'd prefer to have the maximal set of keys in `compute_from`, particularly when we're sidestepping a single key (as suggested by the comment)—we'll see the status change if something alters the story in the broad feature.

On the other hand, maybe https://github.com/web-platform-dx/web-features/pull/1231 will provide us with sufficient visibility into support changes? I'm not sure.

Reply:
I think a minimal set will serve us better than a maximal set and that https://github.com/web-platform-dx/web-features/pull/1231 will indeed add visibility into whether the choice makes sense.

The minimal set is quite easy to determine without checking implementation status, it's just the entrypoint(s) to the feature most of the time.

Reply:
In this case there are more keys that could be listed, like "api.TrustedTypePolicyFactory.createPolicy", but it's hard to say where to draw the line for a minimal set.

Reply:
Yeah, now that I've slept on this I think keying on entrypoints is the right way to go. It suggests a pretty good test for inclusion even, something close to: "omitting such a key would make the feature unusuable."